recipe/meta.yaml

```diff
@@ -102,7 +125,7 @@ outputs:
         - {{ cross_stdlib }}_{{ cross_target_platform }}
         - compiler-rt_{{ cross_target_platform }}
         - compiler-rt =={{ clang_version }}.*
```

Comment:
All of these dependencies should be moved to `clang_impl_linux-64` and `clang_linux-64` should only depend on `clang_impl_linux-64`.

Reply:
I'm assuming the binutils activation should also stay on `clang_linux-64`, not the impl package.